helpers/wrapper/contracts/Wrapper.sol

```diff
@@ -123,14 +122,8 @@ contract Wrapper {
       wethContract.withdraw(_makerAmount);
       // Transfer ether to the user.
```

Comment:
Please add breaks between these comment+line pairs for consistency with the rest of the function.